sky/server/stream_utils.py

```diff
@@ -36,6 +41,18 @@ async def log_streamer(request_id: Optional[str],
                        follow: bool = True) -> AsyncGenerator[str, None]:
     """Streams the logs of a request."""
 
+    buffer: List[str] = []
```

Comment:
minor: might be good to mention why buffer could help here, or maybe include it in the comment for the buffer constants. : )